modules/ding_contact/ding_contact.module

```diff
@@ -47,6 +47,8 @@ function ding_contact_flooded_form($form, $form_state, $message) {
  * Why ? - to make sure no cached page is delivereed
  */
 function ding_contact_form_contact_site_form_alter(&$form, &$form_state) {
+  $message['from'] = variable_get('site_mail');
+  $message['headers']['Sender'] = variable_get('site_name', 'DDB CMS');
```

Comment:
This refers to an undefined `$message` variable. That does not seem right?

Reply:
@kasperg Thank you noticed. Probably just forgot to remove.